core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1482,9 +1481,9 @@ public synchronized void onBlocksDownloaded(Peer peer, Block block, @Nullable Fi
 
         @Override
         public synchronized void run() {
-            if (blocksInLastSecond > 1) {
```

Comment:
Because it's meant to be used for tracking download speed. It doesn't tell you much of any use for a single block.